packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1001,11 +1001,20 @@ export const MissingLocale = {
 		`The locale/path \`${locale}\` does not exist in the configured \`i18n.locales\`.`,
 } satisfies ErrorData;
 
+/**
+ * @docs
+ * @description
+ * Astro could not find the index URL of your website. An index page is required so that Astro can create a redirect from the main index page to the index page of the default locale when using [`i18n.routing.prefixDefaultLocale`](https://docs.astro.build/en/reference/configuration-reference/#i18nroutingprefixdefaultlocale) and [`i18n.routing.redirectToDefaultLocaleSection`](https://docs.astro.build/en/reference/configuration-reference/#i18nroutingredirecttodefaultlocale).
+ * @see
+ * - [Internationalization](https://docs.astro.build/en/guides/internationalization/)
```

Comment:
```suggestion
 * - [Internationalization](https://docs.astro.build/en/guides/internationalization/#routing)
``` 
I'd go straight to this section!